src/Ryujinx.Ui.Common/App/ApplicationLibrary.cs

```diff
@@ -928,6 +928,12 @@ public static (Nca patch, Nca control) GetGameUpdateData(VirtualFileSystem fileS
                 {
                     updatePath = JsonHelper.DeserializeFromFile(titleUpdateMetadataPath, _titleSerializerContext.TitleUpdateMetadata).Selected;
 
+                    if (!string.IsNullOrWhiteSpace(updatePath)
+                        && !File.Exists(updatePath))
```

Comment:
No need of a new line here
```suggestion
                    if (!string.IsNullOrWhiteSpace(updatePath) && !File.Exists(updatePath))
```